components/common/searchBar.tsx

```diff
@@ -0,0 +1,45 @@
+import { SearchIcon } from './searchIcon';
+import * as React from 'react';
+import { withStyles } from '@mui/material';
+import { SvgIcon } from '@mui/material';
+
+/*
+Title: SearchProps
```

Comment:
This front matter (title, last updated, etc.) isn't needed because this is an open source project, and this information can be found using Git natively.

Reply:
Comments have been updated to Doc comments and headers were removed.